api_tests/lib2/wallets/index.ts

```diff
@@ -0,0 +1,85 @@
+import { Asset as AssetSdk, Ledger as LedgerSdk } from "comit-sdk";
+import { HarnessGlobal } from "../../lib/util";
+import { sleep } from "../utils";
+import { BitcoinWallet } from "./bitcoin";
+import { EthereumWallet } from "./ethereum";
+
+declare var global: HarnessGlobal;
+
+interface AllWallets {
+    bitcoin?: BitcoinWallet;
+    ethereum?: EthereumWallet;
+}
+
+export type LedgerKind = keyof AllWallets;
+export type AssetKind = "bitcoin" | "ether" | "erc20";
```

Comment:
```suggestion
export enum AssetKind {
    Bitcoin,
    Ether,
    Erc20
}
```
Why not using an enum?